pandas/io/pickle.py

```diff
@@ -52,15 +81,17 @@ def to_pickle(obj, path, compression='infer', protocol=pkl.HIGHEST_PROTOCOL):
 def read_pickle(path, compression='infer'):
     """
     Load pickled pandas object (or any other pickled object) from the specified
-    file path
+    file path.
```

Comment:
Can you try to fit this on one line? 

Maybe the "or any other pickled object" is not needed in the summary line and can go in an extended summary, as the typical use case should be pandas objects. 
Or, maybe the "from specified file path" can be shortened to "from file."

Reply:
shortened.